src/camvidlog2/cli.py

```
@@ -91,64 +96,49 @@ def query(
-            outpath = outdir / "avg" / f"{rank:03d}.jpg"
-            save(outpath, img_array)
-            del img_array
-
-    # output for each query separately
-    for j, _ in enumerate(embedding_group.items, 0):
-        results = calculate_results(distances[j])
+    distances_index = 0
+    for j, embedding_group in enumerate(embedding_collection.groups):
+        # calculate the average distance over the embeddings in this group
+        group_distances = distances.iloc[
+            :, distances_index : distances_index + len(embedding_group.items) + 1
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)